egs/tedlium/s5_r2/local/chain/run_tdnn.sh

```diff
@@ -149,7 +149,7 @@ if [ $stage -le 18 ]; then
     --chain.leaky-hmm-coefficient 0.1 \
     --chain.l2-regularize 0.00005 \
     --chain.apply-deriv-weights false \
-    --chain.lm-opts="--num-extra-lm-states=2000" \
+    --chain.lm-opts="--ngram-order=5 --num-extra-lm-states=2000" \
```

Comment:
Did you really find that --ngram-order=5 was better?  By how much?
In general I don't like too much tuning that's specific to specific egs directories.  People copy them to other setups, and I prefer to have settings that will work everywhere.